machines/vcItem.ts

```diff
@@ -782,6 +782,20 @@ export const vcItemMachine =
           };
         }),
 
+        logVcPinToggle: send(
+          (context) =>
+            ActivityLogEvents.LOG_ACTIVITY({
+              _vcKey: VC_ITEM_STORE_KEY(context),
+              type: context.isPinned ? 'VC_PIN' : 'VC_UNPIN',
+              timestamp: Date.now(),
+              deviceName: '',
+              vcLabel: context.id,
+            }),
```

Comment:
IMO, the ticket states that the **audit logs** of the specifically selected VC should be shown and not the action of logging the VC being pinned & unpinned. Am I missing a discussion, if so please update the Jira ticket with the newer description.